vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/view/components/ProjectDetailsComponent.java

```diff
@@ -90,24 +122,48 @@ public void setStyles(String... componentStyles) {
    */
   private final class Handler {
 
-    private final ProjectDetailsComponent component;
     private final ProjectInformationService projectInformationService;
+    private final PersonSearchService personSearchService;
 
     private ProjectId selectedProject;
 
-    public Handler(ProjectDetailsComponent component,
-        ProjectInformationService projectInformationService) {
-      this.component = component;
+    public Handler(ProjectInformationService projectInformationService,
+        PersonSearchService personSearchService) {
+
       this.projectInformationService = projectInformationService;
+      this.personSearchService = personSearchService;
+
       setFieldsEditableOnlyOnFocus();
       attachSubmissionActionOnBlur();
       restrictInputLength();
+      setUpPersonSearch(projectManagerComboBox);
+      switchModeForProjectManagerField();
+    }
+
+    private void switchModeForProjectManagerField() {
+      // click focuses the field and hide component
+      projectManagerLayout.addClickListener(click -> {
+        pmElement.setVisible(false);
+        projectManagerComboBox.setVisible(true);
+        projectManagerComboBox.focus();
```

Comment:
Question: By binding it to the layout, this means that the entire width will trigger the combobox selection(See attached movie). Is this the intended behaviour? 

https://user-images.githubusercontent.com/29627977/202168040-4c6718f8-6990-4a19-b04b-d080b9922587.mov